terraform/environments/data-and-insights-wepi/locals.tf

```diff
@@ -1,58 +1,7 @@
-# This data sources allows us to get the Modernisation Platform account information for use elsewhere
-# (when we want to assume a role in the MP, for instance)
-data "aws_organizations_organization" "root_account" {}
-
-# Get the environments file from the main repository
-data "http" "environments_file" {
-  url = "https://raw.githubusercontent.com/ministryofjustice/modernisation-platform/main/environments/${local.application_name}.json"
-}
-
-data "aws_caller_identity" "oidc_session" {
-  provider = aws.oidc-session
-}
-
-data "aws_caller_identity" "modernisation_platform" {
-  provider = aws.modernisation-platform
-}
+#### This file can be used to store locals specific to the member account ####
```

Comment:
when there are already resources in locals, I tend to remove the info heading, as the teams already are using it and therefore do not need informing, but I guess it is just a preference